.travis.yml

```diff
@@ -12,6 +12,8 @@ addons:
 
 language: rust
 jobs:
+    allow_failures:
+          rust: beta
```

Comment:
The standard for YAML is two spaces, not four. Given that this can introduce indentation bugs, we should stick with two spaces.

Reply:
I removed two spaces; does that work?

Reply:
Oh sorry! That wasn't clear. I meant the indentation bumps should all be two spaces. The rest of the file has examples of this. Currently you have them at 4 spaces.

Reply:
okay; it took a few times to see what you meant, but hopefully I got everything.